projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/dimension/scd1/02-handle-quality-checks.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+
+from vdk.api.job_input import IJobInput
+
+SQL_FILES_FOLDER = (
+    os.path.dirname(os.path.abspath(__file__)) + "/02-requisite-sql-scripts"
+)
+
+
+def run(job_input: IJobInput):
+    job_arguments = job_input.get_arguments()
+
+    check = job_arguments.get("check")
+    target_schema = job_arguments.get("target_schema")
+    target_table = job_arguments.get("target_table")
+    staging_schema = job_arguments.get("staging_schema")
+    insert_query = get_query("02-insert-into-target.sql")
+
+    if check:
+        if not staging_schema:
+            raise Exception(
+                "No staging_schema specified to execute the defined data checks against."
+            )
+
+        staging_table = f"{staging_schema}.{target_table}"
+        align_stg_table_with_target(
+            f"{target_schema}.{target_table}", staging_table, job_input
+        )
+
+        insert_into_staging = insert_query.format(
```

Comment:
This probably is not an issue with this PR but my understanding of VDK. 
Here the insert statement is created with target table and schema. 
However how does it know what the source table and scheme is?

Reply:
I expect all the arguments that are not passed explicitly to be inherited from the arguments passed to the template call.